bitnami/openldap/2.5/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -65,7 +65,8 @@ export LDAP_EXTRA_SCHEMAS="${LDAP_EXTRA_SCHEMAS:-cosine,inetorgperson,nis}"
 export LDAP_SKIP_DEFAULT_TREE="${LDAP_SKIP_DEFAULT_TREE:-no}"
 export LDAP_USERS="${LDAP_USERS:-user01,user02}"
 export LDAP_PASSWORDS="${LDAP_PASSWORDS:-bitnami1,bitnami2}"
-export LDAP_USER_DC="${LDAP_USER_DC:-users}"
+export LDAP_USER_OU="${LDAP_USER_OU:-users}"
+export LDAP_GROUP_OU="${LDAP_GROUP_OU:-groups}"
```

Comment:
As in my suggestion, please keep the value `LDAP_USER_DC` but remove its default value.

Then, for `LDAP_USER_OU` and `LDAP_GROUP_OU` it will have the following priorities:
1. Value provided in `LDAP_USER_OU`/`LDAP_GROUP_OU`
2. If no value provided and `LDAP_USER_DC`, use value in `LDAP_USER_DC`.
3. If neither `LDAP_USER_OU`/`LDAP_GROUP_OU` or `LDAP_USER_DC` provided, then use default values `user/group`.

```suggestion
export LDAP_USER_DC="${LDAP_USER_DC:-}"
export LDAP_USER_OU="${LDAP_USER_OU:-${LDAP_USER_DC:-users}}"
export LDAP_GROUP_OU="${LDAP_GROUP_OU:-${LDAP_USER_DC:-groups}}"
```

Then, around line 174, inside `ldap_validate` add the following message:
```
if [[ -n "$LDAP_USER_DC" ]]; then
    warn "The env variable 'LDAP_USER_DC' has been deprecated and will be removed in a future release. Please use 'LDAP_USER_OU' and 'LDAP_GROUP_OU' instead."
fi
```